### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by SaraStephenRaj**

### CODE REVIEW

File: src/test/java/com/scale/stepdefs/CommonAPISteps.java [hunk 1]
```diff
@@ -67,6 +67,14 @@ public void iFirePUTLotSupplierEndpointAndAssertTheStatus() {
...
         apibase.putLotSupplier();
     }
-
+    @When("I fire PUT Agreement endpoint and create {string}")
+    public void iFirePUTAgreementEndpointAndCreate(String agreementNumber) {
+        apibase.getOAUTH2Grant();
+        apibase.putRequestedAgreement(agreementNumber);
+    }
+    @Then("assert {string} is created")
+    public void assertIsCreated(String agreementNumber) {
```
Comment (amrutha-sireesh) [inline src/test/java/com/scale/stepdefs/CommonAPISteps.java:76]: please rename method accordingly eg: IsRMINumberCreated

File: src/test/resources/FeatureFiles/Agreement Service API Tests/Agreement Service API .feature [hunk 2]
```diff
@@ -44,6 +44,7 @@ Feature: Agreement Service API tests
   Scenario:Put Agreement
     When I fire PUT Agreement endpoint and assert the status
```
Comment (amrutha-sireesh) [inline src/test/resources/FeatureFiles/Agreement Service API Tests/Agreement Service API .feature:46]: Remove this Test as the Scenario:PUT an agreement and verify in GET call covers this now

File: src/test/resources/FeatureFiles/Website UI Tests/UpcomingAgreement.feature [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@RegressionTest
+@RegressionTest1
```
Comment (amrutha-sireesh) [inline src/test/resources/FeatureFiles/Website UI Tests/UpcomingAgreement.feature:1]: could you please change this tag back for me?

**Review by amrutha-sireesh — APPROVED** [review]

File: src/test/java/com/scale/stepdefs/CommonAPISteps.java (hunk 1 above)
Comment (SaraStephenRaj) [reply src/test/java/com/scale/stepdefs/CommonAPISteps.java:76]: Renamed

File: src/test/resources/FeatureFiles/Agreement Service API Tests/Agreement Service API .feature (hunk 2 above)
Comment (SaraStephenRaj) [reply src/test/resources/FeatureFiles/Agreement Service API Tests/Agreement Service API .feature:46]: Done

**Pull request merged**